vendor/bat-native-ads/src/bat/ads/internal/ml/pipeline/text_processing/embedding_processing_unittest.cc

```diff
@@ -0,0 +1,76 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ads/internal/ml/pipeline/text_processing/embedding_processing.h"
+
+#include <map>
+#include <utility>
+#include <vector>
+
+#include "base/files/file.h"
+#include "bat/ads/internal/base/unittest/unittest_base.h"
+#include "bat/ads/internal/base/unittest/unittest_file_util.h"
+#include "bat/ads/internal/ml/data/vector_data.h"
+#include "bat/ads/internal/resources/contextual/text_embedding/text_embedding_resource.h"
+
+// npm run test -- brave_unit_tests --filter=BatAds*
+
+namespace {
+
+constexpr char kSimpleResourceFile[] =
+    "wtpwsrqtjxmfdwaymauprezkunxprysm_simple";
```

Comment:
single items inside anonymous namespaces ideally should not have empty lines above/below